clients/src/main/java/org/apache/kafka/clients/producer/KafkaProducer.java

```diff
@@ -1472,7 +1472,7 @@ public boolean isDone() {
     /**
      * Callbacks that are called by the RecordAccumulator append functions:
      *  - user callback
-     *  - interceptor callbacks
+     *  - interceptor callback
```

Comment:
There can be multiple interceptors and that's likely the reason why it says `callbacks` here.